frontend/javascripts/oxalis/shaders/segmentation.glsl.ts

```diff
@@ -271,12 +271,18 @@ export const getSegmentationId: ShaderModule = {
 
   <% _.each(segmentationLayerNames, function(segmentationName, layerIndex) { %>
     vec4[2] getSegmentationId_<%= segmentationName %>(vec3 worldPositionUVW) {
-      vec4[2] volume_color =
+      vec4[2] volume_color;
+      vec3 transformedCoordUVW = transDim((<%= segmentationName %>_transform * vec4(transDim(worldPositionUVW), 1.0)).xyz);
+      if (isOutsideOfBoundingBox(transformedCoordUVW)) {
+        return volume_color;
```

Comment:
`volume_color` is uninitialized at this point, is the returned value deterministic in that case or should we initialize it to be sure?

Reply:
good question! just tested it and it should be safe :)